net/shorewall6/Makefile

```diff
@@ -0,0 +1,114 @@
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_HASH:=3b5c228693936103eb1479e5b0968c5a7d24d94beea410b3ad79f9e7e3b32cca286384b91ee3b09ce006758d27f9b2efe22c589d07d05eeef9ef981818da0b80
+
+PKG_MAINTAINER:=Willem van den Akker <wvdakker@wilsoft.nl>
+PKG_LICENSE:=GPL-2.0+
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/shorewall6
+    SECTION:=net
+    CATEGORY:=Network
+    DEPENDS:=+ip +ip6tables +shorewall-core +perl +perlbase-autoloader +perlbase-autouse +perlbase-dynaloader +perlbase-digest +perlbase-findbin +perlbase-getopt +perlbase-hash
+    TITLE:=Shorewall6 Central Administration System
+    URL:=http://www.shorewall.net/
+    SUBMENU:=Firewall
+endef
+
+define Package/shorewall6/description
+	The Shoreline Firewall, is high-level tool for configuring Netfilter.
+
+	Shorewall allows for central administration of multiple IPv6 firewalls. 
+	This is the full Shorewall product which will compile Shorewall scripts 
+	It is not recommended to run it on a low memory system.
+
+	Note: This is the IPv6 implementation of Shorewall.
+	      This full Shorewal packages also installs Perl which can make the image big (about +2M). 
+endef
+
+CONFIGURE_ARGS += \
+	vendor=openwrt
+
+define Package/shorewall6/conffiles
+/etc/shorewall6/accounting
+/etc/shorewall6/actions
+/etc/shorewall6/blrules
+/etc/shorewall6/clear
+/etc/shorewall6/conntrack
+/etc/shorewall6/findgw
+/etc/shorewall6/hosts
+/etc/shorewall6/init
+/etc/shorewall6/interfaces
+/etc/shorewall6/lib.private
+/etc/shorewall6/maclist
+/etc/shorewall6/mangle
+/etc/shorewall6/nat
+/etc/shorewall6/netmap
+/etc/shorewall6/params
+/etc/shorewall6/policy
+/etc/shorewall6/providers
+/etc/shorewall6/proxyndp
+/etc/shorewall6/refresh
+/etc/shorewall6/refreshed
+/etc/shorewall6/restored
+/etc/shorewall6/routes
+/etc/shorewall6/rtrules
+/etc/shorewall6/rules
+/etc/shorewall6/scfilter
+/etc/shorewall6/secmarks
+/etc/shorewall6/shorewall6.conf
+/etc/shorewall6/snat
+/etc/shorewall6/start
+/etc/shorewall6/started
+/etc/shorewall6/stop
+/etc/shorewall6/stopped
+/etc/shorewall6/stoppedrules
+/etc/shorewall6/sysconfig
+/etc/shorewall6/tcclasses
+/etc/shorewall6/tcclear
+/etc/shorewall6/tcdevices
+/etc/shorewall6/tcfilters
+/etc/shorewall6/tcinterfaces
+/etc/shorewall6/tcpri
+/etc/shorewall6/tunnels
+/etc/shorewall6/zones
+endef
+
+define Build/Compile
+	DESTDIR=$(PKG_INSTALL_DIR) $(PKG_BUILD_DIR)/install.sh
+endef
+
+define Package/shorewall6/install
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_DIR) $(1)/etc/shorewall6
+	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/usr/share/shorewall6
+	$(CP) $(PKG_INSTALL_DIR)/etc/init.d/shorewall6 $(1)/etc/init.d
+	$(CP) $(PKG_INSTALL_DIR)/etc/shorewall6/* $(1)/etc/shorewall6
+	$(INSTALL_BIN) ./files/hotplug_iface $(1)/etc/hotplug.d/iface/05-shorewall6
+	$(INSTALL_BIN) ./files/vardir $(1)/etc/shorewall6/vardir
```

Comment:
If the destination filename is staying the same, you should omit it, i.e.

`$(INSTALL_BIN) ./files/hostname $(1)/etc/shorewall6/`

Reply:
Done